### PR TITLE
Completed jQuery AJAX Diagnostic

### PR DESCRIPTION
**Pull request by pjliddy**
Comfort: 5
Clarity: 5

I used the examples we created in class as templates to answer these questions.

### CODE REVIEW

File: diagnostic.md [hunk 1]
```diff
@@ -51,13 +61,21 @@ Write the `curl` request you'd use to delete a single donut, using whatever
 ID you'd like.
 
 ```sh
-# your answer here
+curl 'http://www.example.com/donuts/$ID' \
```
Comment (benjimelito) [inline diagnostic.md]: URL should go at the end: `curl --include --request DELETE http://www.example.com/donuts/$ID`

Comment (benjimelito) [inline diagnostic.md:64]: URL should go at the end: `curl --include --request DELETE http://www.example.com/donuts/$ID`

File: diagnostic.md [hunk 2]
```diff
@@ -73,14 +91,31 @@ data in JSON format.
 ```
 
 ```sh
-# your answer here
+curl  http://www.example.com/donuts/ \
```
Comment (benjimelito) [inline diagnostic.md:94]: Same as above ^^

**Comment by benjimelito** [comment]
Nice, 8/10 👍

File: diagnostic.md (hunk 2 above)
Comment (pjliddy) [reply diagnostic.md:94]: Just An FYI, these two examples are verbatim from the repos we have from the API talks. The only thing I modified is the base URL string. In the examples, one line curl scripts put the URL at the end, but multi line scripts put the URL on the first line after curl but before the first backslash.

**Pull request closed without merging**